src/routes/rules/RulesView.vue

```diff
@@ -132,13 +127,11 @@
             role="img"
           />
           <h1 class="gradient-text">
-            One-Offs
+            {{ t('rules.oneoffs.title') }}
```

Comment:
It looks like this is broken
![image](https://github.com/cuttle-cards/cuttle/assets/6520704/0ec86336-6bd4-46aa-a0e1-205aa5efde41)

Reply:
Oh no, I'll take a look

Reply:
No problem, this happens. And the translations are something that we've struggled to test in a way that balances flexibility to change the text with the accuracy that the text is actually rendered. I have an idea for how to address it but it's not fully baked. I'll bring it up for discussion in the discord and then we can write up an issue to upgrade our testing of translations in a separate PR

Reply:
Solved in [f392376](https://github.com/cuttle-cards/cuttle/pull/685/commits/f39237642e935a34ec17a5cff416ca5faa606d32)